### DISCUSSION TITLE
sonoff pow r2 - kWh LED

### DISCUSSION THREAD

**Discussion by petrhod** [arendst/Tasmota#20357] [Q&A]
I have sonoff pow r2 with Tasmota. I need to set the LED output (approx. 100ms) every time 0.01 KWh is subtracted from the consumption (100 pulses to kWh)
Is this possible ?
Thank you in advance.

**Comment by sfromis** [comment]
You could try something like using rules to detect if the value is 0.01 larger than the prior. Configuring the led as a relay gives you direct control via the Power command and associated options, like PulseTime.

**Comment by petrhod** [comment]
I have to save the current data from "EnergyTotal" to VAR, and then search for a difference of 0.01 kWh. If there is a match, will the reaction and new save be updated in VAR?
It's too complicated for me....
